packages/elements-semantic/src/Tag/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC } from 'react';
-import { TagProps, Theme } from '@frontegg/react-core';
-import { Icon, Label, LabelProps as SemanticLabelProps, SemanticCOLORS } from 'semantic-ui-react';
+import { TagProps, Icon, Theme } from '@frontegg/react-core';
```

Comment:
if you want to use Icon in `elements-semantic` should be imported from '../Icon' from the same library.